system/security/shared/caching.cpp

```diff
@@ -556,6 +556,7 @@ bool CPermissionsCache::queryPermsManagedFileScope(ISecUser& sec_user, const cha
     if (!fullScope || !*fullScope)
     {
         *accessFlags = queryDefaultPermission(sec_user);
+        WARNLOG("FileScope unspecified for %s, applying default permissions %s(%d), took %dms", sec_user.getName(), getSecAccessFlagName(*accessFlags), *accessFlags,  msTick()-start);
```

Comment:
Who is the audience for this warning?  I would have expected the operator, rather than the user. - in which case OWARNLOG() is more appropriate.  Same question for the other logging statements.